### ISSUE TITLE
Where is this portal "https://skynet.moe" for?

### ISSUE THREAD

**Issue by parajbs** [SkynetLabs/skynet-kernel#255]
Hello **David Vorick and developers,**

I have a question:
You use the portal "https://skynet.moe" in the code here.
I tested the portal in the browser and saw that not all skylinks can be called up.

Only a skylink with a **jpg file worked**.
`https://skynet.moe/CADd2MjdgMbxOW4-5zJRo_SRr5nZyTLZh_LzEM-sIo8TIw`

Now the question
**which content types can all be called up?**

errors:
Content type "text/html" not allowed.
Content type "application/pdf" not allowed.
Content type "application/zip" not allowed.
Content type "audio/mpeg" not allowed.
Content type "audio/wav" not allowed.

**Where is this portal "https://skynet.moe" for???**

**What does this sentence mean?**
`This service serves user-generated content from the Sia Skynet network (https://skynetlabs.com/about). The concept is very similar to an IPFS gateway (https://docs.ipfs.tech/concepts/ipfs-gateway/).`

**Comment by pcfreak30** [comment]
The portal is operated by @redsolver and runs on fly.io. It is intended for small files only and acts as a proxy. Since the kernel tries multiple portals, the fact that it fails on large files does not matter.

**Comment by redsolver** [comment]
Yep, the portal is very new. It has a very restricted list of allowed content types right now, to prevent the domain being blocked for phishing for example. Large files are not blocked, if their content type is allowed a redirect to siasky.net is returned. skynet.moe also uses an additional CDN layer to make the delivery of small allowed files as fast as possible.

**Comment by redsolver** [comment]
I will likely add more allowed content types in the future, for example audio.
